### PR TITLE
Big band musical instrument update

### PR DESCRIPTION
**Pull request by JustSumBody**
Adds ~~six~~ seven new musical instruments: A trombone, glockenspiel, accordion, recorder, saxophone, harmonica, and stationary xylophone. Sprites courtesy of Kranenberg(hope I spelled that right)

Adds two new cargo crates: A random instrument crate for 50 credits that contains any random instrument and a Big band crate that contains all of the instruments for 500 credits. Prices/restrictions are irrelevant to me feel free to argue over it

Removed Guitar from arts and crafts crate and restored it to its default price.

Some instruments such as the guitar, accordion, and glockenspiel must now be two handed. Arguably all of them should be, but these are necessary because the sprites do not work well with two hands. Accordions and glocks need to be picked up with two hands, but guitars can be wielded instead.

Guitars can now be placed on your back, and have their own one-handed sprites so they don't clash as much
image

Moved instrument sounds to their own folder

Videos showing it all off(Warning: DANGEROUSLY MIDI)
Recorder(ys3 shop theme)
https://dl.dropboxusercontent.com/s/wdpevn7fi9rti4s/2016-08-19_01-32-10.mp4
Accordion(Remove Kebab)
https://dl.dropboxusercontent.com/s/01d5hzqcof6o7ov/2016-08-19_01-32-52.mp4
glockenspiel(maplestory login)
https://dl.dropboxusercontent.com/s/vnic1wk74e2v1wy/2016-08-19_01-33-30.mp4
saxophone(careless whispers)
https://dl.dropboxusercontent.com/s/jn5kpfrej6wqne6/2016-08-19_01-34-15.mp4
trombone(76 trombones)
https://dl.dropboxusercontent.com/s/2ifyuvv14p77tqa/2016-08-19_01-35-08.mp4
xylophone(1812 overture)
https://dl.dropboxusercontent.com/s/mqqpyf340d01fc9/2016-08-19_01-35-53.mp4
Harmonica(oh suzanna) ~~(Sprites pending)~~ now with sprites
https://dl.dropboxusercontent.com/s/xj1mbdrx6pf8j6s/2016-08-19_09-11-38.mp4
Trombone and sax sound meh but the rest aren't too bad I don't think.

### CODE REVIEW

**Comment by nuklearcellphoneg** [comment]
Hello, please add a harmonica.

Literally the same instrument as an accordion, but it fits in your pocket and is lung powered.

**Comment by nuklearcellphoneg** [comment]
harmonica

**Comment by Kurfursten** [comment]
That accordion song... what's that from?

I swear it's going to be like the generic RPGmaker shop song isn't it?

**Comment by JustSumBody** [comment]
It's remove kebab

**Comment by Kurfursten** [comment]
Now I remember. This was one of the themes used in /v/irginquest and it played incessantly for like the first 15 minutes of the game.

**Comment by unid15** [comment]
Holy shit

:+1:

File: code/game/objects/items/devices/instruments.dm [hunk 1]
```diff
@@ -96,4 +108,14 @@
...
 	icon_state = "recorder"
 	item_state = "recorder"
-	instrumentId = "recorder"
\ No newline at end of file
+	instrumentId = "recorder"
+	
+/obj/item/device/instrument/harmonica
+	name = "harmonica"
+	desc = "For when you get a bad case of the space blues."
+	icon_state = "guitar"
```
Comment (Kurfursten) [inline code/game/objects/items/devices/instruments.dm]: ?

Comment (JustSumBody) [inline code/game/objects/items/devices/instruments.dm]: Pending sprites, don't worry I'll change it when I get them

**Comment by SkowronX** [comment]
damn fine work

**Comment by ScrambledSteak** [comment]
When you said "Big Band Music" I thought you were gonna let me dress up like Big Band and punch nerds with my giant brass fists.

this is different than what I was expecting but it's only slightly disappointing

good work

**Comment by jknpj** [comment]
Drums/Bass when?

**Comment by ComicIronic** [comment]
The saxophone inclusion is god tier. Can't wait for the bartender to start a cigar club with a jazz band in the background.

**Comment by FudgePucker** [comment]
> accordion

Hahaha yes

**Comment by Shadowmech88** [comment]
image
One man band instrument when?

**Comment by Megamaw** [comment]
that's a bad equipped harmonica sprite. i'm p. sure someone made a better one earlier

**Comment by clusterfack** [comment]
There isn't much code to review here, as long as you didn't break inhands again I think this will be fine :^)

**Comment by SkowronX** [comment]
Aw ye

**Pull request merged**